ts/screens/authentication/components/PictogramOnSmallDevice.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  IOColors,
+  IOPictograms,
+  IOPictogramSizeScale,
+  IOStyles,
+  Pictogram
+} from "@pagopa/io-app-design-system";
+import React from "react";
+import { Dimensions, View } from "react-native";
+
+export const MIN_HEIGHT_TO_SHOW_FULL_RENDER = 820;
+
+type IOPictogramsProps = {
+  name: IOPictograms;
+  color?: IOColors;
+  pictogramStyle?: "default" | "light-content" | "dark-content";
+  size?: IOPictogramSizeScale | "100%";
+};
```

Comment:
We can easily get rid of the entire block using `ComponentProps<typeof Pictogram>`, better if directly applied to the component (not using a redundant `IOPictogramProps` type)
```suggestion
type IOPictogramsProps = ComponentProps<typeof Pictogram>
```

Reply:
Good suggestion, thank you very much!